### ISSUE TITLE
Update Lesson 10 to new format

### ISSUE THREAD

**Issue by thoward** [sensu/sensu-enterprise-workshop#92]
Lesson 10 outline:

Title: Silencing & Scheduled Maintenance
- Goals
- Silencing alerts
    - TL;DR… users often use this as a sort of “acknowledgement”
    - Exercise: silence an alert 
        1. Add check to monitor the demo app w/ 10s interval (HTTP)
        1. Add `is_silenced` filter to your alert handler (Sensu doesn’t assume that silencing should have the same effect on every handler)
        1. TODO: remove the occurrence filter; add a blockquote comment to explain that we’re only doing this to observe the effect of the silence for this exercise
        1. Invoke demo app failure (curl -XPOST localhost:9000/healthz)
        1. Create a silence using `sensuctl silenced create --interactive` with begin:now, expiry:3600, expire_on_resolve:false
        1. Observe the effect of the silence
        1. Restore demo app to healthy status (curl -XPOST localhost:9000/healthz)
- How to	 schedule maintenance 
    - TL;DR… 
    - Exercise: configure a silence that starts and ends in the future
        1. Create a silence using `sensuctl silenced create --interactive` with begin:180, expire:-1, expire_on_resolve:false
        1. Invoke demo app failure 
        1. Wait / observe maintenance mode
        1. Restore demo app to healthy status 
- Discussion
    - Recap
    - Automated maintenance mode… use Sensu API to create/remove silence at beginning/end of configuration management
